src/node.ts

```diff
@@ -27,11 +33,13 @@ class Node extends VEventTarget {
   /**
    * @param name  See {@link name}
    */
-  constructor(name: string, network?: Network) {
+  constructor(name: string, network?: Network, nodeType?: NodeType) {
```

Comment:
This is not a pretty pattern. This NodeType object just serves to process an existing Node object, it's a weird approach where a class that extends Node should be used instead. But I know that in this particular PR this solution just complies with the present infrastructure.